frontend/src/__tests__/cypress/cypress/e2e/projects/ProjectList.cy.ts

```diff
@@ -0,0 +1,161 @@
+  it('should start with an empty project list', () => {
+    initIntercepts();
+    initEmptyProjectIntercepts();
+    cy.visitWithLogin('/projects');
+    projectListPage.shouldBeEmpty();
+  });
+
+  it('should open a modal to create a project', () => {
+    initIntercepts();
+    initEmptyProjectIntercepts();
+    cy.visitWithLogin('/projects');
+    projectListPage.findCreateProjectButton().click();
+    createProjectModal.shouldBeOpen();
+    createProjectModal.findCancelButton().click();
+    createProjectModal.shouldBeOpen(false);
+    projectListPage.findCreateProjectButton().click();
+    createProjectModal.shouldBeOpen();
+    createProjectModal.findSubmitButton().should('be.disabled');
+  });
+
+  it('should create project', () => {
+    initIntercepts();
+    initCreateProjectIntercepts();
+
+    projectListPage.visit();
+    projectListPage.navigate();
+    projectListPage.findCreateProjectButton().click();
+    createProjectModal.findNameInput().type('My Test Project');
+    createProjectModal.findDescriptionInput().type('Test project description.');
+    createProjectModal.findSubmitButton().should('be.enabled');
+    createProjectModal.findResourceNameInput().should('have.value', 'my-test-project').clear();
+    createProjectModal.findResourceNameInput().should('have.attr', 'aria-invalid', 'true');
+    createProjectModal.findSubmitButton().should('be.disabled');
+    createProjectModal.findResourceNameInput().type('test-project');
+
+    cy.intercept(
+      '/api/k8s/apis/project.openshift.io/v1/projects',
+      mockK8sResourceList([mockProjectK8sResource({})]),
+    ).as('createdProjects');
+
+    createProjectModal.findSubmitButton().click();
+    cy.wait('@createdProjects');
+    cy.url().should('include', '/projects/test-project');
+  });
+
+  it('should list the new project', () => {
+    initIntercepts();
+    cy.intercept(
+      '/api/k8s/apis/project.openshift.io/v1/projects',
+      mockK8sResourceList([mockProjectK8sResource({})]),
+    );
+    projectListPage.visit();
+    projectListPage.navigate();
+    projectListPage.clickProjectScopeDropdown().should('exist');
+    projectListPage.findDatascienceProjectOption('Data science projects').should('be.visible');
+    projectListPage
+      .findDatascienceProjectOption(
+        'Only projects created in the Data Science Dashboard are displayed.',
+      )
+      .should('be.visible');
+    projectListPage.findAllprojectOption('All available projects').should('be.visible');
+    projectListPage
+      .findAllprojectOption(
+        'All available projects created in the Data Science Dashboard and OpenShift are displayed.',
+      )
+      .should('be.visible');
+    projectListPage.clickProjectScopeDropdown();
+    projectListPage.findCreateProjectButton().should('exist');
+    projectListPage.findJupyterLauncher().should('exist');
+    projectListPage.shouldHaveProjects();
+    projectListPage.findProjectFilterDropdown().should('exist');
```

Comment:
We have some custom commands called `findDropdownItem` and `findSelectOption`. 
Would those work here?

Reply:
Thanks @christianvogt for the suggestion.We can use custom command here